client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -281,6 +290,49 @@ class TransferOtherUser extends React.Component {
 		);
 	}
 
+	renderTransferInformation() {
+		const { isMapping, selectedDomainName: domainName, selectedSite, translate } = this.props;
+
+		if ( isMapping ) {
+			return (
+				<>
+					<p>
+						{ translate(
+							'Please choose an administrator to transfer domain mapping of {{strong}}%(domainName)s{{/strong}} to.',
+							{ args: { domainName }, components: { strong: <strong /> } }
+						) }
+					</p>
+					<p>
+						{ translate(
+							'You can transfer this domain mapping to any administrator on this site. If the user you want to ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'You can transfer this domain to any administrator on this site. If the user you want to transfer is not currently an administrator, please {{a}}add them to the site first{{/a}}.' )` *ES Score: 8*